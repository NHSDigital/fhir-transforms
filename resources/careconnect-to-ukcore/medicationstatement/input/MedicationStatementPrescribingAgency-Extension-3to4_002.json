{"resourceType":"MedicationStatement","extension":[{"url":"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-PrescribingAgency-1","valueCodeableConcept":{"coding":[{"system":"https://fhir.nhs.uk/STU3/CodeSystem/CareConnect-PrescribingAgency-1","code":"prescribed-by-another-organisation","display":"Prescribed by another organisation"}],"text":"Hospital"}}]}
