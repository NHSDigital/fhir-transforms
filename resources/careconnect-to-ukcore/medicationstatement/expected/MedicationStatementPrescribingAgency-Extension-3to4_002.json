{
  "resourceType": "MedicationStatement",
  "extension": [
    {
      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-MedicationPrescribingOrganization",
      "valueCodeableConcept": {
        "coding": [
          {
            "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-MedicationPrescribingOrganization",
            "code": "prescribed-by-another-organisation",
            "display": "Prescribed by another organisation"
          }
        ],
        "text": "Hospital"
      }
    }
  ]
}